### PR TITLE
Gdm test scenarios

### PR DESCRIPTION
**Pull request by ggbecker**
Create test scenarios for:

- gdm dconf rules (the ones who uses gdm.d directory)
- dconf_db_up_to_date

Added rule to desired profiles.

### CODE REVIEW

File: tests/data/group_system/group_software/group_gnome/dconf_test_functions.sh [hunk 1]
```diff
@@ -4,6 +4,11 @@ clean_dconf_settings(){
 	rm -rf /etc/dconf/db/*
 }
```
Comment (matejak) [inline tests/data/group_system/group_software/group_gnome/dconf_test_functions.sh:17]: Could you please add a function `install_dconf_and_gdm_if_needed` and call it from tests? That way, copy-pasting will be reduced and tests will be more readable as well.

**Review by matejak — CHANGES_REQUESTED** [review]
Thank you for the PR, I have just one remark about copy-pasted code.

File: tests/data/group_system/group_software/group_gnome/dconf_test_functions.sh (hunk 1 above)
Comment (ggbecker) [reply tests/data/group_system/group_software/group_gnome/dconf_test_functions.sh:17]: Done

**Comment by matejak** [comment]
Great job, thanks!

**Review by matejak — APPROVED** [review]

**Pull request merged**